RustBoy/UI/RustBoyView.swift

```diff
@@ -96,3 +119,20 @@ internal struct RustBoyView: View {
         return documentsPath + romFilename + ".sav"
     }
 }
+
+struct RustBoyView_Preview: PreviewProvider {
```

Comment:
What is this?

Reply:
This is how you make a SwiftUI preview that you can use for development. It is ignored in release builds.

Reply:
So this shows how the UI will look for the listed devices.